dart/constraint/ConstraintSolver.cpp

```diff
@@ -478,6 +479,29 @@ void ConstraintSolver::updateConstraints()
   // Destroy previous soft contact constraints
   mSoftContactConstraints.clear();
 
+  // Create a mapping of contact pairs to the number of contacts between them
+  using ContactPair
+      = std::pair<collision::CollisionObject*, collision::CollisionObject*>;
+
+  // Compare contact pairs while ignoring their order in the pair.
+  struct ContactPairCompare
+  {
+    ContactPair SortedPair(const ContactPair& a) const
```

Comment:
Nit: Please rename the function name to `getSortedPair` to follow the function name convention

Reply:
https://github.com/dartsim/dart/pull/1505/commits/5d81fbbc3a61d493f7f4f201657aa482ed169d2c